workflow.rst

```diff
@@ -493,5 +500,5 @@ Learn more
 .. toctree::
    :maxdepth: 1
 
-   workflow/introduction
-   workflow/dumping-workflows
+   /workflow/workflow-and-state-machine
```

Comment:
This change could be reverted, but it's stange to have an introduction that is not main entry point. More over there are no links at the end of the page.